scripts/catalog-tests/generate-tdr-snapshot-metadata.py

```diff
@@ -0,0 +1,85 @@
+#------------------------------------------------------------------------------
+# Generate TDR Snapshot Metadata
+#
+# This script adds snapshots to the catalog as the test user from DC-218,
+# pulling the metadata from the resources JSON file
+# 
+# Before Running:
+#   python3 -m pip install requests argparse
```

Comment:
```suggestion
#   python3 -m venv .env
#   source .env/bin/activate
#   pip install --upgrade pip
#   python3 -m pip install requests argparse
```

I suggest installing all the package in a virtualenv, `pip install` is brittle and can break your Python installation which can be difficult to fix on MacOS :(

Reply:
ok! :)